locales/fr-FR.json

```diff
@@ -11,5 +11,7 @@
   "serverNotFound": "Serveur introuvable",
   "settings": "Paramètres",
   "unexpectedError": "Erreur inattendue",
-  "usernameRequired": "Nom d'utilisateur requis"
+  "usernameRequired": "Nom d'utilisateur requis",
+  "libraryNotFound": "Bibliothèque non trouvée",
```

Comment:
"Bibliothèque" implies books.

I think "Médiathèque" would fit better here (It's what is used by jf-web currently).

Reply:
Also "Médiathèque introuvable" sounds slightly better, I think?

Not too sure what sounds the most natural for an error message, since I end up writing and reading more English than French on a daily basis...

Reply:
Nah you're right, I wasn't too happy with "non trouvée" haha.

Reply:
Done